webrender/Cargo.toml

```diff
@@ -32,6 +32,8 @@ ws = { optional = true, version = "0.7.3" }
 serde_json = { optional = true, version = "1.0" }
 serde = { optional = true, version = "1.0" }
 serde_derive = { optional = true, version = "1.0" }
+image = { optional = true, version = "*" }
```

Comment:
We should probably specify a version here?

Reply:
@nc4rrillo Are you able to specify a version number here? Then we can get this merged :)